upload/library/Sidane/Threadmarks/Install.php

```diff
@@ -111,4 +103,60 @@ public static function uninstall()
     $db->delete('xf_permission_entry_content', "permission_id = 'sidane_tm_menu_limit'");
     $db->delete('xf_permission_entry_content', "permission_id = 'sidane_tm_view'");
   }
+
+  public static function modifyColumn($table, $column, $oldDefinition, $definition)
+  {
+    $db = XenForo_Application::get('db');
+    $hasColumn = false;
+    if (empty($oldDefinition))
+    {
+      $hasColumn = !$db->fetchRow('SHOW COLUMNS FROM `'.$table.'` WHERE Field = ?', $column);
```

Comment:
The return value of `fetchRow` shouldn't be inverted here as `$hasColumn` will be `false` when the column exists and the column will not be modified.

When I upgraded the add-on with this change, `label` remained as a `varchar(100)`.